### ISSUE TITLE
Fix Build errors

### ISSUE THREAD

**Issue by just-ary27** [OWASP/BLT-Flutter#275]
Most of the workflows are stuck rn, most likely due to this error. It would be good if closed asap.
![image](https://user-images.githubusercontent.com/76696648/227119846-9c951311-3494-4de0-92b9-41c30def324f.png)

**Comment by letsintegreat** [comment]
This is probably because in this workflow run, the flutter SDK downloaded was for Arm64 architecture, but we need it for x64 architecture. I just tried running the [same workflow on my fork](https://github.com/letsintegreat/BLT-Flutter/actions/runs/4497713900/jobs/7913572119), and it downloaded the correct SDK. So, I am guessing they made changes in [subosito/flutter-action](https://github.com/subosito/flutter-action)? Anyways, can you please trigger the workflow again and see if it goes through now?

**Comment by letsintegreat** [comment]
Yep, here it is: https://github.com/subosito/flutter-action/issues/219
Pretty sure the workflow will go through in the next trigger.

**Comment by letsintegreat** [comment]
This issue can be closed now.
